backend/src/impl/auth.py

```diff
@@ -8,7 +8,7 @@
 from flask import current_app, g
 
 
-def check_ApiKeyAuth(user_email: str, api_key: str, required_scopes):
+def check_api_key_auth(user_email: str, api_key: str, required_scopes):
```

Comment:
These two functions are called [here](https://github.com/neulab/explainaboard_web/blob/371d2780103f425518d11d9d90d48a8157198c75/backend/templates/authorization_controller.mustache#L10) so the name is determined by `openapi.yaml`. Do you think we can skip these two functions here? We can also update `openapi.yaml` or the template to snake case.

Reply:
@lyuyangh Oh, I didn't know the OpenAPI definition is related. Is `ApiKeyAuth` also related to https://github.com/neulab/explainaboard_web/blob/371d2780103f425518d11d9d90d48a8157198c75/openapi/openapi.yaml#L502?

Reply:
Yep.
```
{{#authMethods}}
{{#isApiKey}}
def check_{{name}}(api_key, required_scopes):
    return auth.check_{{name}}(api_key, required_scopes)
```
This is from `authorization_controller.mustache` which generates `authorization_controller.py`. It gets the `name` from `openapi.yaml`. That's why it is called "ApiKeyAuth".

Reply:
@lyuyangh I see. Thanks for the explanation. I'll revert this change and add comments to skip the two functions. I think no change in `openapi.yaml` or the template is required at this point. Changing naming conventions itself is a topic we need a discussion separately. I'm not familiar with the naming convention and best practices yet.

Reply:
Reverted the changes, and added comments why we want to silence flake8 for the two functions.